dirs/dirs.go

```diff
@@ -235,6 +236,7 @@ func SetRootDir(rootdir string) {
 	SnapNamesFile = filepath.Join(SnapCacheDir, "names")
 	SnapSectionsFile = filepath.Join(SnapCacheDir, "sections")
 	SnapCommandsDB = filepath.Join(SnapCacheDir, "commands.db")
+	SnapExtraInfoDir = filepath.Join(SnapCacheDir, "extra")
```

Comment:
If you follow my suggestion to use more descriptive entry this could become `"store"`